<?xml version="1.0"?>

<launch>
  <arg name="watchdog_enabled" default="false" />

	<node pkg="taurob_base_node" type="taurob_base_node" name="taurob_base_node" output="screen">
		<param name="publish_tf" value="true" />
		<param name="tf_from_frame" value="/base_link" />
		<param name="tf_to_frame" value="/odom" />
		<param name="watchdog" value="$(arg watchdog_enabled)" />
 		<param name="control_enabled_at_startup" value="true" />
		<param name="base_ip" value="10.0.0.121" />
		<param name="base_port" value="9080" />

	</node>
	
</launch>
